ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_QMLSamples/android/AndroidManifest.xml

```diff
@@ -14,7 +14,7 @@
 
     <!-- %%INSERT_FEATURES -->
     <supports-screens android:anyDensity="true" android:largeScreens="true" android:normalScreens="true" android:smallScreens="true"/>
-    <application android:name="org.qtproject.qt.android.bindings.QtApplication" android:extractNativeLibs="true" android:hardwareAccelerated="true" android:label="Qt Samples - QML" android:allowBackup="true" android:fullBackupOnly="false" android:icon="@drawable/icon">
+    <application android:name="org.qtproject.qt.android.bindings.QtApplication" android:extractNativeLibs="true" android:hardwareAccelerated="true" android:label="Qt Samples - QML" android:allowBackup="true" android:allowNativeHeapPointerTagging="false" android:fullBackupOnly="false" android:icon="@drawable/icon">
```

Comment:
Same change should be applied to C++ sample app viewer.

Reply:
done